cmd/podman/play/kube.go

```diff
@@ -118,9 +118,11 @@ func init() {
 
 		buildFlagName := "build"
 		flags.BoolVar(&kubeOptions.Build, buildFlagName, false, "Build all images in a YAML (given Containerfiles exist)")
-	}
 
-	if !registry.IsRemote() {
```

Comment:
You're allowing SignaturePolicy for `podman-remote` now - is that expected?

Reply:
No this just joined the out block of if !registry.IsRemote()